npm/npm.go

```diff
@@ -313,3 +315,16 @@ func NewNetworkPolicyManager(clientset *kubernetes.Clientset, informerFactory in
 
 	return npMgr
 }
+
+// reconcileChains checks for ordering of AZURE-NPM chain in FORWARD chain periodically.
+func (npMgr *NetworkPolicyManager) reconcileChains() {
+	iptMgr := iptm.NewIptablesManager()
+	var err error
+	for {
+		time.Sleep(reconcileChainTimeInMinutes * time.Minute)
+
+		if err = iptMgr.CheckAndAddForwardChain(); err != nil {
+			metrics.SendErrorLogAndMetric(util.NpmID, "Error: failed to add AZURE-NPM chain to FORWARD chain.")
```

Comment:
the specific error context is lost here, better to add it to the message?

Reply:
Removing and returning this error as the log message will be redundant